perfmetrics/scripts/ml_tests/tf/resnet/setup_host_and_run_model.sh

```diff
@@ -6,7 +6,8 @@ set -e
 cd "$HOME/github/gcsfuse/perfmetrics/scripts"
 
 echo "Setting up the machine with Docker and Nvidia Driver..."
-source ml_tests/setup_host.sh
+DRIVER_VERSION="450.172.01"
```

Comment:
Ohh, this was missed.

Learning: Always check the dependent code if we are changing the interface of any commonly used script/code.

Reply:
Sure